config/samples/nvidia_v1alpha1_nvidiadriver.yaml

```diff
@@ -6,9 +6,9 @@ spec:
   # use pre-compiled packages for NVIDIA driver installation.
   usePrecompiled: false
   driverType: gpu
-  repository: nvcr.io/nvidia
+  repository: "1837582943" #nvcr.io/nvidia
```

Comment:
please revert this, this was required only for testing using your private image.